packages/fds-components/src/components/buttons/GroupButton.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+export const GroupButton = (props) => {
+  const { className, active, Icon, children, ...rest } = props;
+
+  const rootClass = cx('group-btn', className, {
+    active,
+  });
+
+  return (
+    <button className={rootClass} {...rest}>
+      {Icon && <Icon size="xs" className="margin--right--half" />}
+      {children}
```

Comment:
Using `children` for the label provides more formatting flexibility for the label.
Using a `label` prop would give the component more control/restriction over how the label is displayed.

One nice thing about restricting this down to a `label: PropTypes.string` prop is that we don't have to account for accidental nesting.

Reply:
Two questions:

1. If there was a label prop., would we allow children?
2. What would you think about blending the two strategies, and requiring children to be a string? There's arguments against that, but wondering what you think.

Reply:
I've gone ahead and opted for label, following our "lock it down by default" sort of strategy.

Reply:
But yeah I think it's a strong idea 👍

Reply:
I feel like it's more of an all or nothing decision - accept any children, or accept only one primitive to lock it down (string, number, etc).

One more argument _for_ using a dedicated prop instead of children is that props are self-documenting with prop types. It's nice to give hints.